spec/rubocop/cop/rspec/nested_groups_spec.rb

```diff
@@ -22,6 +22,22 @@
     RUBY
   end
 
+  it 'support --auto-gen-config' do
+    expect_offense(<<-RUBY)
```

Comment:
This expectation exists already in line 7. You can just use `inspect_source` here instead of `expect_offense`.